code/src/components/HappyPage.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react'
+
+import HappyForm from './HappyForm'
+import HappyList from './HappyList'
+
+const HappyPage = () => {
+
+    const [isLoading, setIsLoading] = useState(false)
+    const [thoughtList, setThoughtList] = useState([])
+    const [newThought, setNewThought] = useState('')
+
+    const HAPPY_THOUGHTS_API = 'https://happy-thoughts-technigo.herokuapp.com/thoughts/'
+
+    const getHappyThougths = () => {
+        setIsLoading(true)
+        fetch(HAPPY_THOUGHTS_API)
+            .then((response) => response.json())
+            .then((data) => setThoughtList(data))
+            .catch((error) => console.error(error))
+            .finally(() => setIsLoading(false))
+    }
+
+    useEffect(() => {
+        getHappyThougths()
+    }, [])
+
+
+    const postNewThought = () => {
```

Comment:
When I post a new thought, maybe the text input field should be cleared, so I see that something happened and I can write something else in the text field, without having to erase the previous text?